y-octo/src/doc/codec/item.rs

```diff
@@ -118,16 +118,15 @@ pub(crate) struct Item {
     pub id: Id,
     pub origin_left_id: Option<Id>,
     pub origin_right_id: Option<Id>,
-    #[cfg_attr(all(test, not(loom)), proptest(value = "Somr::default()"))]
-    pub left: Somr<Item>,
-    #[cfg_attr(all(test, not(loom)), proptest(value = "Somr::default()"))]
-    pub right: Somr<Item>,
+    #[cfg_attr(all(test, not(loom)), proptest(value = "Somr::none()"))]
+    pub left: ItemRef,
+    #[cfg_attr(all(test, not(loom)), proptest(value = "Somr::none()"))]
+    pub right: ItemRef,
     pub parent: Option<Parent>,
+    // TODO: Optmize
+    //   use Box<str> and share it with same parent_sub items
```

Comment:
consider https://github.com/servo/string-cache ?

Reply:
string-cache is too much for this simple scenario and I don't think too much on this optimization, let's ignore it now.